models/human.rb

```diff
@@ -1,29 +1,21 @@
 class Human
 
+  attr_accessor :job_title, :age, :drinking_age
+  attr_reader :name
+
   def initialize value
     @name = value
   end
 
-  def name
-    @name
-  end
-
   def legal_name
-    first_name = @name.split.first
-    last_name = @name.split.last
-
-    last_name + ", " + first_name
-
-    # @name.split(' ', 2).reverse.join(', ')
+    @name.split().reverse.join(', ')
   end
 
-  # attr_accessor :job_title
-  def job_title=(value)
-    @job_title = value
-  end
+  #@drinking_age = 18
 
-  def job_title
-    @job_title
+  def can_drink?
+    @drinking_age ||= 18
```

Comment:
might this be a situation where having `def drinking_age` getter may be better and we default it there ensuring any other method that uses it defaults to the same value.
ps. same for `age`